README.md

```diff
@@ -33,89 +33,92 @@ System.Web.Bem - БЭМ-инфрастурктура для ASP.NET MVC.
+### Specificity of BEM projects
+[BEM](https://en.bem.info) (Block-Element-Modifier) is the frontend development methodology, which was created in [Yandex](https://yandex.com/company). It is based on the component approach. BEM is also the set of tools for comfortable development according to principles of methodology. BEM hepls to develop the sites faster and support them for a long time.
 
-Согласно правилам БЭМ, приложение состоит из независимых блоков, которые лежат в отдельных папках. Каждый блок реализован в нескольких технологиях (шаблоны, стили, клиентский код). Чтобы код блоков мог работать в приложении, блоки собирают в бандлы.
+According to the BEM principles, application consists of independent blocks, which located in separate folders. Each block is implemented in several technologies (templates, styles, client-side code). Blocks should be assembled into the bundles to use them in the running application.
 
-Декларация бандла - файл с перечислением блоков, которые должны попасть в бандл. На основе декларации сборщик собирает бандл, учитывая зависимости блоков и уровни переопределения. Бандл собирается отдельно для каждой технологии. Во время работы приложения бандл шаблонов используется для формирования html (на сервере и клиенте), бандлы js и css подключаются на страницы и используются на клиенте.
+Bundle declaration is a file with the list of blocks, which should be in a bundle. On the base of declaration assembler will build a bundles according to dependencies of blocks and levels of redifinition. Bundle is built separately for each technology. During the application is running the bundle of templates is used for html generation (on the server and the client side), js bundles and style bundles are included in the pages and used on client side.
 
-**System.Web.Bem** - БЭМ-инфрастурктура для ASP.NET MVC проектов. При установке в проект [NuGet пакета System.Web.Bem](https://www.nuget.org/packages/System.Web.Bem/):
-- добавляется папка `Bem` с файловой структурой БЭМ проекта;
-- из npm ставится сборщик БЭМ-проектов [enb](https://ru.bem.info/toolbox/enb/) и настройки для сборки БЭМ-бандлов при компиляции ASP.NET MVC проекта;
-- подключается C# библиотека `System.Web.Bem` для серверной шаблонизации во время работы приложения.
+**System.Web.Bem** is the BEM-infrastructure for ASP.NET MVC projects. During the installation of [System.Web.Bem package](https://www.nuget.org/packages/System.Web.Bem) into the project:
+- folder `BEM` with the files structure is created;
+- assembler of BEM-projects [enb](https://ru.bem.info/toolbox/enb) is installed from npm and proper configuration file for enb is created;
```

Comment:
-> The [enb](https://ru.bem.info/toolbox/enb) assembler for BEM projects is installed from npm, and the enb configuration file is created.